mlir/lib/Dialect/Rock/utility/transformMapUtils.cpp

```diff
@@ -2071,13 +2241,14 @@ mlir::rock::removeUpperDims(OpBuilder &b, ArrayAttr transformAttrs,
     llvm::SmallVector<int64_t> lowerBounds = {};
     FailureOr<rock::TransformMapAttr> maybeNewTrMapAttr =
         removeUpperDimsFromMap(b, trMap, removeIndicesSet, upperBounds,
-                               lowerBounds);
+                               lowerBounds, preservedSubDims);
```

Comment:
it looks like "preservedSubDims" is not used here, can't it be an internval variable of removeUpperDimsFromMap?

Reply:
its used between calls same as "removeIndicesSet".